LMSAdmin/src/main/java/com/lms/LMSAdmin/controller/AuthorController.java

```diff
@@ -15,8 +20,51 @@
 	@Autowired
 	AuthorService authorService;
 	
-	@RequestMapping(value = "/admin/author/{authorId}", method=RequestMethod.POST)
-	public Author getAuthorById(@PathVariable int authorId) {
+	//test
+	@RequestMapping("/lmsAdmin/author/{authorId}")
+	public Author getAuthorById(@PathVariable int authorId) throws SQLException {
 		return authorService.getAuthorById(authorId);
 	}
+	
+	/*@RequestMapping("/lmsAdmin/menu/1/authMenu/{authOpt}")
+	public void authorMenu(@PathVariable int authOpt) throws SQLException {
+		switch(authOpt) {
+			case 1:
+				insertAuthor();
+ 			case 4: 
+				getAllAuthors();
+				break;
+			default:
+				
+				break;
+		}
+	}*/
+	

```

Comment:
Since you're unsure if the code is needed for future reason, have them copied and pasted to another branch repository so that you can have proper code readability because if you never came back to this part, then there is a chance you might forget to come back and remove it.